src/engraving/dom/edit.cpp

```diff
@@ -2179,7 +2179,12 @@ void Score::cmdFlip()
     };
 
     for (EngravingItem* e : el) {
-        if (e->isNote() || e->isStem() || e->isHook()) {
+        if (e->hasVoiceApplicationProperties()) {
+            flipOnce(e, [e]() {
+                PlacementV curPlacement = e->getProperty(Pid::PLACEMENT).value<PlacementV>();
+                e->undoChangeProperty(Pid::DIRECTION, curPlacement == PlacementV::ABOVE ? DirectionV::DOWN : DirectionV::UP);
```

Comment:
I haven't checked all precious PRs so I'm a bit surprised that you need to check the _placement_ and then set the _direction_ based on that, but I assume that's alright :)

Reply:
Before the new voicing options, these items only had the Placement property, which only provides two values (above/below). Now they needed an Auto option, hence why I am using the Direction property (which provides Auto/Up/Down). The direction property is the one that is shown to (and controlled by) the user, an in turn the placement property (which holds the actual above/below state) is determined depending on the direction. Hence why, to do a flip, I need to check the placement, but set the direction :)